Sources/FisherYates/random.swift

```diff
@@ -8,7 +8,7 @@
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
 
-#if os(OSX)
+#if os(macOS)
```

Comment:
Was “macOS” back-ported to older versions of Swift, or do we need to keep it as “OSX” for compatibility?

Reply:
Hmm, I didn't think about that. I'll leave it alone, then.